src/blockchains/eos/utils/logger.ts

```diff
@@ -0,0 +1,42 @@
+interface ILoggerOptions {
+  channels: string[]
+}
+
+function getLogChannels(conf?: string): string[] {
+  if (!conf) {
+    return []
+  }
+
+  if (conf === "*") {
+    return ["ERR", "WARNING", "INFO", "LOG"]
+  }
+
+  return conf.split(",")
+}
+
+const getOptions = (): ILoggerOptions => ({
+  channels: getLogChannels(process.env.DEBUG)
+})
+
+function genericLog(channel: string, message?: any, ...optionalParams: any[]) {
```

Comment:
We should have a discussion about using custom logger vs [winston](https://github.com/winstonjs/winston)/[bunyan](https://github.com/trentm/node-bunyan)/whatever.

Reply:
It will be not so hard to change in the future